### PR TITLE
Manually fixed new headers in ext package

### PR DESCRIPTION
**Pull request by d-michail**
I corrected a little bit the headers in the ext package and also added the new header in the grammar files.

### CODE REVIEW

**Comment by jkinable** [comment]
Thanks for the corrections. jGraphT at the moment has over 400 .java files, so it was impossible to correct all headers by hand. Unfortunately, there were so many inconsistencies in the headers that it was impossible to capture them all consistently in a single script. An example of this can be found in GraphMLExporterTest.java in this pull request. The 2nd copyright header states:
(C) Copyright 2003-2008, by Barak Naveh and Contributors.
However, the code also contains:
Original Author:  Trevor Harmon
and
@author Trevor Harmon

Obviously, in this case the 2nd copyright header should have stated:
(C) Copyright 2003-2008, by Trevor Harmon and Contributors.
So the script wrongly assigned copyright to Barak Naveh.

Unfortunately, there are also files where the reverse is true:

(C) Copyright 2003-2008, by Some Company or Organization and Contributors.
@author Some Person

Here it would be wrong to assign the copyright to the author, as it would overwrite the company's copyright.

So by default, the script copies the names in the 2nd copyright header if present. If absent, the script searches for @author, or Original Author annotations.

In summary, I expect that there are more inconsistencies of this sort, so it's probably a good habit to check the correctness of each header every time we modify some existing class.

TLDR: check the copyright notice in all your files
